guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,1149 @@
+
+
+#### Image Quality Verification during Image Acquisition {#image-quality-verification-during-image-acquisition}
+
+Help the radiology technician improve their results.  During an image acquisition a QA score is provided to the radiology technician to help validate if the image quality is sufficient and if imaging needs to be re-taken.  Typically seconds or 10’s of seconds matter for efficient, high quality imaging.
+
+**Clinical Workflow**
+
+ 
+
+
+
+1. A patient gets scheduled for a routine imaging for a CT imaging.
+2. During Image acquisition time, the radiology technician is watching the imaging proceed and parts of the study be completed from his computer.
+3. As parts of the study complete, they get sent to an AI system for image quality verification.  The modality sends out parts of the DICOM study, one series at a time to an AI system.  Size of each series 512x512x100.  (CT, taking this is seconds, for MRI this is minutes). The modality is connected to an AI processing system via DICOM with a 1 gbps network connection. 
+4. AI system analyses the arriving series images with a volumetric  classification algorithm to notice if the patient is moving.  AI system creates a DICOM SR, including results.
+5. If the AI system notices any anomalies in the image it The AI system sends the results (DICOM SR) to two locations, PACS and radiographers desktop computer a notification to the radiology technician about the Quality Score of the image. 
+6. While monitoring the imaging the technician receives the notification about image quality score onto their desktop computer.  The technician notices the image QA score not being acceptable and decides to restart the imaging series. 
+
+**Clinical Use Case:** Image Quality Verification
+
+**Computational Workload Type:** [Asynchronous Result](#asynchronous-result-computational-workload)
+
+**Clinical workload:** 1 request per series, Example: 1600 requests for 512x512x1600 CT series image.  20100 CT studies per day per modality, 3 series in each study.  60 requests per individual  modality per day, 600 requests per day in the whole clinic .  &lt;610 seconds from image sent to modality to response being available to technician. 
+
+**Importance to Clinical AI Deployment:** medium 
+
+The implementation of AI as part of image quality will not directly impact patient care, but does improve quality of hospital operations by reducing patient recall.
+
+**Figure:** Image Quality Validation
+
+![MONAI_Image QA at Acquistion](assets/MONAI_Image_QA_Acquistion.jpg)
+
+
+### AI Diagnostics {#ai-diagnostics}
+
+
+#### Emergency DiagnosticsDiagnosis {#emergency-diagnosticsdiagnosis}
+
+Immediate diagnostics required for emergency care: immediate processing is required for a clinical diagnosis of the trauma. These workloads have the patient either awaiting further imaging or diagnostics. Seconds count for some emergency diagnostics situations (such as stroke).
+
+**Clinical Workflow**
```

Comment:
should this be H5 (aka `#####`) instead of strong/bold?

Reply:
Strong bold.  You can look at the first use case and see the same strong/bold with two different examples. If i change this one, I'd have to change them all.  (I'm an Engineer not an English major) Again, feel free to pull the branch and consistently set all the bold clinical workflows to an H5 header.